make_jar

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env ruby
+require 'fileutils'
+
+# Remove previously compiled file
+FileUtils.rm_f("bofrev.jar")
+
+# Generate executable
+system("warble jar")
 
-system("mkdir bin")
```

Comment:
@panmari: as far as I know (from the warble README), the assume that the exectuable script (bofrev) is located within a directory called bin. this is why i run mkdir, copied bofrev the bofrev script into this folder, and after the process, performed the cleanup statge.